collections/misc/occurrencesearch.php

```diff
@@ -121,23 +124,23 @@ function isNumeric(inStr){
 						</div>
 					</div>
 					<div style="clear:both;padding:2px;">
-						<div style="float:left;width:130px;">Catalog #:</div>
+						<div style="float:left;width:130px;"><?php echo $LANG['CATALOG']; ?> #:</div>
```

Comment:
Since # might not mean number in all languages, I would include # in this language tag. Same for Other Catalog #